src/core/ext/filters/http/client_authority_filter.cc

```diff
@@ -71,7 +71,7 @@ ArenaPromise<TrailingMetadata> ClientAuthorityFilter::MakeCallPromise(
 
 namespace {
 const grpc_channel_filter grpc_client_authority_filter =
-    MakePromiseBasedFilter<ClientAuthorityFilter>();
+    MakePromiseBasedFilter<ClientAuthorityFilter, true>("authority");
```

Comment:
How about defining an enum for `kIsClient` instead of a bool, so that call sites are self-documenting w.r.t. this parameter?